app/models/studio.rb

```diff
@@ -1,3 +1,8 @@
 class Studio < ApplicationRecord
   has_many :movies
+  has_many :actors, through: :movies
+
+  def actor_list
+    actors.distinct.pluck(:name).join(", ")
```

Comment:
Adding `.join` here is a gray area MVC-wise -- we can argue that it's valuable to put in the model if we want to granularly test it at a unit level, but folks may also say that it is more display logic and should live in the view. 

As long as you unit test it, my tendency is to err on the side of encapsulating it in a model method, but wanted to let you know that it's a gray area! Just make sure you do test it :)